.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: go
+
+go:
+  - 1.8
+  - 1.9
+  - 1.10
+  - 1.11
```

Comment:
I think adding `tip` is useful to check next version Go does not break this library.

Reply:
Sounds good!